eng/pipelines/templates/steps/template-test-run-job.yml

```diff
@@ -100,6 +86,13 @@ steps:
               AZD_DEBUG_FORCE_NO_TTY="1"
           subFolder: "$(Build.SourcesDirectory)/temp"
           runCmd: |
+              # Uninstall azd version
+              curl -fsSL https://aka.ms/uninstall-azd.sh | bash
+
+              # Install azd from PR
+              curl -fsSL https://aka.ms/install-azd.sh | bash -s -- --version $(AzdVersion) --verbose
```

Comment:
This won't work for `pr`, right? Still better than what we have currently though

Reply:
Yes, just switching to daily for now.